### PR TITLE
feat(publishing): add support for CocoaPods

### PR DESCRIPTION
**Pull request by wqz-leo**
We'll need to get rid of version extensions in the release tags in order to publish this SDK to CocoaPods trunk.

### CODE REVIEW

File: Mtribes.podspec [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Pod::Spec.new do |mtribes|
+
+  mtribes.name         = "Mtribes"
+  mtribes.version      = "1.0.0"
+  mtribes.summary      = "mtribes is a new platform that brings together UX control, user targeting and audience insights under one centralized console."
+
+  mtribes.homepage     = "https://mtribes.com/"
+
+  mtribes.license      = { :type => "Apache License, Version 2.0", :file => "LICENSE.txt" }
```
Comment (mspmax) [inline Mtribes.podspec:9]: this should be our license right?

File: Mtribes.podspec [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  mtribes.author       = { "Deltatre" => "Register@deltatre.com" }
```
Comment (mspmax) [inline Mtribes.podspec:11]: should be probably mtribes related

**Comment by mspmax** [comment]
> We'll need to get rid of version extensions in the release tags in order to publish this SDK to CocoaPods trunk.

Does this mean we are not doing manual installations anymore?

File: Mtribes.podspec (hunk 1 above)
Comment (wqz-leo) [reply Mtribes.podspec:9]: You mean the license generated by the CLI?

File: Mtribes.podspec (hunk 2 above)
Comment (wqz-leo) [reply Mtribes.podspec:11]: Brian said we can use this email address, not sure if there's an email for mtribes.

**Comment by wqz-leo** [comment]
> > We'll need to get rid of version extensions in the release tags in order to publish this SDK to CocoaPods trunk.
> 
> Does this mean we are not doing manual installations anymore?

Manual installation is still an option. Developers can choose either way to install the library.

File: Mtribes.podspec (hunk 1 above)
Comment (mspmax) [reply Mtribes.podspec:9]: yeah, that's our distribution license

Comment (wqz-leo) [reply Mtribes.podspec:9]: Done

File: Mtribes.podspec [hunk 3]
```diff
@@ -0,0 +1,23 @@
+Pod::Spec.new do |mtribes|
+
+  mtribes.name         = "Mtribes"
+  mtribes.version      = "1.0.0"
```
Comment (mspmax) [inline Mtribes.podspec]: is this library version ? if yes, this should be `0.1.0`

**Pull request merged**